### DISCUSSION TITLE
[本地部署]: 无法正常使用

### DISCUSSION THREAD

**Discussion by niuniu857** [GaiZhenbiao/ChuanhuChatGPT#539] [Q&A]
### 是否已存在现有反馈与解答？

- [X] 我确认没有已有issue或discussion，且已阅读**常见问题**。

### 是否是一个代理配置相关的疑问？

- [X] 我确认这不是一个代理配置相关的疑问。

### 错误描述

![image](https://user-images.githubusercontent.com/113335896/229502470-428a57ce-fbd0-4e87-8c4f-2c220399f4b8.png)

### 复现操作

1.正常部署
2.弹出页面出现error

### 错误日志

```shell
Could not create share link. Please check your internet connection or our status page: https://status.gradio.app
```

### 运行环境

```markdown
- OS: windows11
- Browser: chrome
- Gradio version:3.22.1 
- Python version: 3.9.16
```

### 补充说明

无

**Comment by Sk1pper953** [comment]
我也出现了这样的情况

**Comment by Gary2018X** [comment]
这个我是localhost没问题 服务器部署有问题

**Comment by onenov** [comment]
同，服务器部署出现此问题

**Comment by GaiZhenbiao** [comment]
看readme，配置一下Nginx反代

**Comment by LaoJiuYes** [comment]
我用宝塔配置的代理总是error，用ip访问就没事
